openshift/containers/exodus-gw/Dockerfile

```
@@ -0,0 +1,32 @@
+FROM centos:8
+
+# Add sources
+COPY . /usr/local/src/exodus-gw/
+
+# Install extra requirements specific to container deployment
+RUN \
+    # Enable EPEL for certain deps
+    yum -y install epel-release \
+    # Install extra commands needed for build
+    && yum -y install python3 \
+    && yum -y groupinstall 'development tools' \
+    && cd /usr/local/src/exodus-gw \
+    # Install application itself
+    && pip3 install --upgrade pip \
+    && pip3 install --require-hashes -r requirements.txt \
+    && pip3 install --no-deps . \
+    # Clean up unneeded packages
+    && yum -y erase rpm-build python2-pip \
+    && yum -y autoremove \
+    # Clean up unnecessary data
+    && yum clean all && rm -rf /var/yum/cache && rm -rf /usr/local/src/exodus-gw
+
+# Run as a non-root user
+RUN adduser exodus-gw
+USER exodus-gw
+
+# Use 8080 is the default port used by uvicorn
+EXPOSE 8080
+
+# Run the application
+CMD ["uvicorn", "exodus_gw.gateway:app", "--host", "0.0.0.0", "--port", "8080"]
```

Could we use ENTRYPOINT for this instead?

Right now, with CMD, we can't really add extra arguments to this command while running, and I think we'll want to be able to do that.

Also can we please use exodus_gw:application?  It's a bit more stable, it doesn't depend on the fact that we have a python module named "gateway" which seems to be an implementation detail we'll probably change.  Note: this will require a rebase.

```suggestion
ENTRYPOINT ["uvicorn", "exodus_gw:application", "--host", "0.0.0.0", "--port", "8080"]
```